docs/application/native/guides/multimedia/media-controller.md

```diff
@@ -875,6 +887,76 @@ To process the received search command on the server side, follow these steps:
 > This feature supports Tizen 5.0 and Higher for Mobile and Wearable.
 
 
+## Updating and Retrieving Abilities
+To update the abilities on the server side:
+
+1. Create the media controller server handle using the `mc_server_create()`:
+
+   ```
+   ret = mc_server_create(&g_server_h);
+   ```
+   
+2. Set the abilities using the corresponding 'mc_server_set_XXX_ability()', or 'mc_server_set_ability_support()':
+   For example, to update the playback ability, set the ability for each playback actions using the 'mc_server_set_playback_ability()', and then update the abilities using the 'mc_server_update_playback_ability()':
+   ```
+   ret = mc_server_set_playback_ability(g_mc_server, MC_PLAYBACK_ACTION_PLAY, MC_ABILITY_SUPPORTED_YES);
+	 ret = mc_server_set_playback_ability(g_mc_server, MC_PLAYBACK_ACTION_PAUSE, MC_ABILITY_SUPPORTED_NO);
+	 ret = mc_server_set_playback_ability(g_mc_server, MC_PLAYBACK_ACTION_STOP, MC_ABILITY_SUPPORTED_YES);
+	 ret = mc_server_set_playback_ability(g_mc_server, MC_PLAYBACK_ACTION_NEXT, MC_ABILITY_SUPPORTED_NO);
+	 ret = mc_server_set_playback_ability(g_mc_server, MC_PLAYBACK_ACTION_PREV, MC_ABILITY_SUPPORTED_YES);
+	 ret = mc_server_set_playback_ability(g_mc_server, MC_PLAYBACK_ACTION_FAST_FORWARD, MC_ABILITY_SUPPORTED_NO);
+	 ret = mc_server_set_playback_ability(g_mc_server, MC_PLAYBACK_ACTION_REWIND, MC_ABILITY_SUPPORTED_YES);
+	 ret = mc_server_set_playback_ability(g_mc_server, MC_PLAYBACK_ACTION_TOGGLE_PLAY_PAUSE, MC_ABILITY_SUPPORTED_NO);
+   
+   ret = mc_server_update_playback_ability(g_mc_server);
+   ```
+   
+   For example, to update the display mode ability, set the ability using the 'mc_server_set_display_mode_ability()':
+   ```
+   ret = mc_server_set_display_mode_ability(g_mc_server, MC_DISPLAY_MODE_LETTER_BOX | MC_DISPLAY_MODE_ORIGIN_SIZE | MC_DISPLAY_MODE_FULL_SCREEN | MC_DISPLAY_MODE_CROPPED_FULL, MC_ABILITY_SUPPORTED_YES);
+   ```
+   
+   For example, to update the display rotation ability, set the ability using the 'mc_server_set_display_rotation_ability()':
+    ```
+   ret = mc_server_set_display_rotation_ability(g_mc_server, MC_DISPLAY_ROTATION_NONE | MC_DISPLAY_ROTATION_90 | MC_DISPLAY_ROTATION_180 | MC_DISPLAY_ROTATION_270, MC_ABILITY_SUPPORTED_YES);
+   ```
+   
+   For other abilities, to update the ability, set the ability using the 'mc_server_set_ability_support()':
+   For example, to update shuffle and repeat ability, set the ability using the 'mc_server_set_ability_support()':
```

Comment:
To update shuffle and repeat ability, set the ability using 'mc_server_set_ability_support()':

Reply:
shuffle and repeat is one of the example for "mc_server_set_ability_support()".
Is it OK to remove "for example"?

Reply:
Yes @haejeongkim 
Global: Suggestion: How about removing "for example":
If you agree, we need to add the following line after line after 924.

The following example demonstrates setting up various abilities: